packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1348,11 +1348,6 @@ export default class TradeStore extends BaseStore {
 
             if (has_currency_changed && should_reset_stake) {
                 obj_new_values.amount = obj_new_values.amount || getMinPayout(obj_new_values.currency ?? '');
-                if (this.is_dtrader_v2_enabled)
-                    this.setV2ParamsInitialValues({
```

Comment:
Removed `stake` from `setV2ParamsInitialValues` as now we are not setting new stake values to trade store before user save the results.